### ISSUE TITLE
I have no change when training on MOSE dataset ,but why have this warning

### ISSUE THREAD

**Issue by iAsakiT3T** [facebookresearch/sam2#492]
UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [64, 256, 1, 1], strides() = [256, 1, 256, 256]
bucket_view.sizes() = [64, 256, 1, 1], strides() = [256, 1, 1, 1] (Triggered internally at ../torch/csrc/distributed/c10d/reducer.cpp:327.)
  return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass

**Comment by lamoboos223** [comment]
what is your GPU? @iAsakiT3T

**Comment by ronghanghu** [comment]
Hi, this warning can be ignore -- it shouldn't affect your training/fine-tuning runs
